src/core/server/graph/mutators/Comments.ts

```diff
@@ -35,6 +36,7 @@ import {
   GQLUnfeatureCommentInput,
 } from "coral-server/graph/schema/__generated__/types";
 
+import { hasTag } from "coral-server/models/comment";
```

Comment:
Can you please move this import up with the others from `coral-server`?

Reply:
And here I thought TS sort imports was magic... I have now found its kryptonite. Yep, I'll fix this up!